cloudflare.go

```diff
@@ -15,6 +15,7 @@ const apiURL = "https://api.cloudflare.com/client/v4"
 
 // API holds the configuration for the current API client. A client should not
 // be modified concurrently.
+const errRequestNotSuccessfull = "Error reported by API"
```

Comment:
You should probably add this to `errors.go` when you are rebasing. You also have an "L" to much at the end of the constant name.